ulakbus/services/common/document_render.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+from zato.server.service import Service
+
+
+class RenderDocuments(Service):
+    def handle(self):
+        self.logger.info('Request: {}'.format(self.request.payload))
+        self.logger.info('Request-Type: {}'.format(type(self.request.payload)))
+
```

Comment:
remove unnecessary log lines, never log user data.. you can add operational logs such as `document render service called`